sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/SearchAsyncClient.java

```diff
@@ -792,8 +792,9 @@ SuggestPagedFlux suggest(String searchText, String suggesterName, SuggestOptions
      * @param suggesterName suggester name
      * @return auto complete result.
      */
+    @ServiceMethod(returns = ReturnType.COLLECTION)
```

Comment:
Why weren't these caught in checkstyle check before? Are we suppressing some valid checks?

Reply:
I remembered checkstyle treated the SearchPagedFlux as Single return type.

Reply:
It should still have an annotation and should only be a suppression in checkstyle xml. I still don't understand why the annotation was completely missing.

Reply:
I added the annotation for all APIs but got checkstyle error for the paging. Had a conversation with Alan, and decided to go without the annotation and add them back until checkstyle gets fixed.
FYI, the error message of the checkstyle is not exactly right as well.